product.py

```diff
@@ -105,6 +113,12 @@ def validate(cls, products):
                     raise UserError(gettext('agronomics.msg_variety_limit',
                     product=product.rec_name))
 
+    def get_alcohol_volume(self, name):
+        if self.template.capacity and self.wine_alcohol_content:
+            return Decimal(
+                (float(self.template.capacity) * float(self.wine_alcohol_content))
+                    / 100).quantize(Decimal(str(10 ** -2)))
```

Comment:
I think it's better to use:

-self.__class__.alcohol_volume.digits[1]

Instead of hardcoding "-2"